kernel/src/process.rs

```diff
@@ -214,6 +236,46 @@ pub trait Process {
     /// kernel-internal errors.
     fn enqueue_task(&self, task: Task) -> Result<(), ErrorCode>;
 
+    /// Enqueue a `Task` to execute the init function of the process.
+    /// The process must be in the the `Terminated` or `Unstarted`
+    /// state, and invoking this method transitions it to the
+    /// `Yielded` state before enqueuing the task. This is the only
+    /// method that transitions a process from `Terminated` or
+    /// `Unstarted` to `Yielded`. Because starting a process has
+    /// security implications (e.g., that every running process has a
+    /// unique application identifier), this method requires a
+    /// Capability.
+    fn enqueue_init_task(
```

Comment:
What happens if this fails?

Reply:
The process doesn't start.

Reply:
Clarified.

Reply:
Did this comment change? What are the errors? Can it be called multiple times? Is the process supposed to be in a yielded state?